apps/communications/contacts/js/views/form.js

```diff
@@ -326,10 +326,12 @@ contacts.Form = (function() {
     currentContact = contact;
     deleteContactButton.parentNode.classList.remove('hide');
     formTitle.setAttribute('data-l10n-id', 'editContact');
+    formHeader.setAttribute('in-edit-mode', 'true');
```

Comment:
you don't need to save 'state' in the DOM element, use global ``'mode'`` variable instead